03_instructional_team/markdown_slides/slides_01.md

```diff
@@ -94,11 +96,11 @@ $ echo "Data Sciences Institute"
 # Course Tools
 
 - [DB Browser for SQLite](https://sqlitebrowser.org/dl/): *Where we will write code*
-- [GitHub](https://github.com/UofT-DSI/sql): *Module overview*
-- [Etherpad](https://pad.riseup.net/p/SQL_DSI_Sept2024): *Where we will keep track of session progress*
+- [GitHub](https://github.com/UofT-DSI/sql): *Course Overview*
```

Comment:
"Module Overview"

Reply:
I switched all Module to "Course" or "Session". Do you want it to say Module here?

Reply:
done